### DISCUSSION TITLE
How to group dates by week in my times?

### DISCUSSION THREAD

**Discussion by pylearn2030** [kimai/kimai#4715] [Questions & Answers]
Dear,
I want to group dates by week in my timesheets. How can I do this ?
Could you please help?
Many thanks!
![image](https://github.com/kimai/kimai/assets/162970815/d72fcad0-6003-4245-94ec-8a41a2e2c9e9)

**Comment by kevinpapst** [comment]
There is no group-by-week view in "My times".

**Reply by pylearn2030** [reply]
Thank you for your quick repsonse. 
What plugin can be used to customize this? 
I want to display group-by-week in both "My times" and "All times".
Thank you!

**Reply by kevinpapst** [reply]
I don't know what that means. I don't know how that should look like.

I assume you come from a paid time-tracker that offers such a feature? 
Which one and how did it look like?

**Reply by pylearn2030** [reply]
Yes. I want to buy a plugin to customize this feature in my Kimai.
